docs/userguide.rst

```diff
@@ -0,0 +1,73 @@
+.. _userguide:
+
+User guide
+==========
+
+This user guide is meant for researchers who would like to structure their data
+analysis and run them on REANA cloud.
+
+REANA file
+----------
+
+It is advised to structure your analysis code using the "inputs", "code",
+"outputs", "environments", "workflows" strategy mentioned in :ref:`concepts`.
+
+REANA client
+------------
+
+REANA is coming with a convenience ``reana-client`` script that you can install
+using ``pip``, for example:
+
+.. code-block:: console
+
+   $ mkvirtualenv reana-client -p /usr/bin/python2.7
+   $ pip install reana-client
+
+You can run ``reana-client --help`` to obtain help.
+
+There are several convenient environment variables you can set when working with
+``reana-client``:
+
+- ``REANA_SERVER_URL`` Permits to specify to which REANA cloud instance the
+  client should connect. For example:
+
+.. code-block:: console
+
+   $ export REANA_SERVER_URL=http://reana.cern.ch
+
+- ``REANA_WORKON`` Permits to specify a concrete workflow ID run for the given
+  analysis. (As an alternative to specifying ``--workflow-id`` in commands.) For
```

Comment:
currently it is called `--workflow` see [here](https://github.com/reanahub/reana-client/blob/master/reana_client/cli/workflow.py#L176) should we open an issue for renaming or we change it here?

Reply:
Amending to ``-workflow`` here and in ``reana-client`` as wel.